index.html

```diff
@@ -359,7 +359,16 @@ <h2 style="text-align:left; margin: 100px 5px 0px 5px">News</h2>
                       <div class="card-body text-center">
                         <h5 class="card-title">Documentation</h5>
                         <p class="card-text">Read about installation, features, tutorials, and more.</p>
-                        <a href="https://posydon.org/POSYDON/latest/index.html" class="btn btn-secondary" target="_blank">Read the docs</a>
+                        <div class="btn-group">
+                          <button type="button" class="btn btn-secondary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            Read the docs
+                          </button>
+                          <div class="dropdown-menu">
+                            <a class="dropdown-item" href="https://posydon.org/POSYDON/latest/index.html" target="_blank">development</a>

```

Comment:
The latest is not development. The latest is the last release: `v2.0.0-pre2` currently